SmartDeviceLink/SDLMutableDataQueue.h

```diff
@@ -0,0 +1,38 @@
+//
+//  SDLMutableDataQueue.h
+//  
+//
+//  Created by David Switzer on 3/2/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface SDLMutableDataQueue : NSObject
+
+NS_ASSUME_NONNULL_BEGIN
+
+// frontBuffer returns the NSMutableData * buffer at the front of the queue,
+// but does not remove it -- modeled after the STL C++ queue front method
```

Comment:
Methods should use API documentation methods. In Xcode 8+, put the cursor on the method and use `cmd+option+/` to generate some helper comments.